GitUp/Application/AppDelegate.m

```diff
@@ -613,6 +620,10 @@ - (IBAction)cloneRepository:(id)sender {
   }
 }
 
+- (IBAction)cloneRepository:(id)sender {
+  [self _cloneRepositoryFromURLString:@""];
```

Comment:
My thought was there was still the option of whether the user wants to pull in submodules. I could change it to just assume yes if you'd prefer.